can't use np.expand_dims with specified keyword argument `axis`

I got the following error when I was using `np.expand_dims`:

```
CRITICAL: I am in trouble. Your input file does not seem to match Pythran's constraints...
test_pythran_funcs.py:5:8 error: function uses an unknown (or unsupported) keyword argument `axis`
----
    b = np.expand_dims(a, axis=1)
        ^~~~ (o_0)
----
```
The codes are:
```
import numpy as np

#pythran export func(float[:])
def func(a):
    b = np.expand_dims(a, axis=1)
    return b
```

If I changed to `b = np.expand_dims(a, 1)`, it just works fine.

--- a/pythran/tables.py
+++ b/pythran/tables.py
@@ -1,7 +1,7 @@
 """ This modules provides the translation tables from python to c++. """
 
 import gast as ast
-from inspect import getfullargspec
+import inspect
 import logging
 import sys
 
@@ -148,7 +148,7 @@ _complex_signature = Union[
 # workaround changes in numpy interaction with getfullargspec
 try:
     import numpy
-    getfullargspec(numpy.asarray)
+    inspect.getfullargspec(numpy.asarray)
     # if we have a description, honor it
     extra_numpy_asarray_descr = {}
 except TypeError:
@@ -4552,12 +4552,9 @@ def save_arguments(module_name, elements):
                 obj = getattr(themodule, elem)
                 while hasattr(obj, '__wrapped__'):
                     obj = obj.__wrapped__
-                spec = getfullargspec(obj)
-                if signature.args.args:
-                    logger.warning(
-                        "Overriding pythran description with argspec "
-                        "information for: {}".format(
-                            ".".join(module_name + (elem,))))
+
+                # first try to gather info through getfullargspec
+                spec = inspect.getfullargspec(obj)
 
                 args = [ast.Name(arg, ast.Param(), None, None)
                         for arg in spec.args]
@@ -4566,9 +4563,16 @@ def save_arguments(module_name, elements):
                          for arg in spec.kwonlyargs]
                 defaults += [spec.kwonlydefaults[kw] for kw in spec.kwonlyargs]
 
+                # Sanity check
+                if signature.args.args:
+                    logger.warning(
+                        "Overriding pythran description with argspec "
+                        "information for: {}".format(
+
+                            ".".join(module_name + (elem,))))
                 # Avoid use of comprehension to fill "as much args/defauls" as
                 # possible
-                signature.args.args = args[:-len(defaults)]
+                signature.args.args = args[:-len(defaults) or None]
                 signature.args.defaults = []
                 for arg, value in zip(args[-len(defaults):], defaults):
                     signature.args.defaults.append(to_ast(value))

--- a/pythran/tests/test_named_parameters.py
+++ b/pythran/tests/test_named_parameters.py
@@ -53,6 +53,13 @@ def call_with_default_and_named_arguments(n, m):
                 return numpy.ones(n, dtype=numpy.uint8).nbytes""",
                       4, intrinsic_named_argument=[int])
 
+    def test_intrinsic_named_argument_without_default(self):
+        self.run_test("""
+            def intrinsic_named_argument_without_default(n):
+                import numpy as np
+                return np.expand_dims(np.ones(n), axis=0)""",
+                      4, intrinsic_named_argument_without_default=[int])
+
     def test_nested_function_with_named_arguments(self):
         self.run_test('''
 def nested_function_with_named_arguments(a):

